packages/jest/src/index.js

```diff
@@ -9,11 +9,11 @@ module.exports = {
 
   rules: {
     'jest/consistent-test-it': ['error', { fn: 'it' }],
-    'jest/lowercase-name': ['error', { ignore: ['describe'] }],
     'jest/no-duplicate-hooks': 'error',
     'jest/no-if': 'error',
     'jest/no-test-return-statement': 'error',
     'jest/prefer-hooks-on-top': 'error',
+    'jest/prefer-lowercase-title': ['error', { ignore: ['describe'] }],
```

Comment:
This was renamed (see https://github.com/jest-community/eslint-plugin-jest/issues/950), but I only caught it when running `eslint` with the latest release of this config and the latest Jest plugin. We should add a test of some kind to CI that catches rules that no longer exist.

Reply:
https://github.com/MetaMask/eslint-config/issues/229